### PR TITLE
Third person follower view controller

### PR DESCRIPTION
**Pull request by otim**
This new view controller follows a target frame. In contrast to XYOrdbit the camera also turns with the target frame. This could be handy if you are doing 3D mapping of a hallway with corners for example.

### CODE REVIEW

**Comment by wjwwood** [comment]
Thanks for the pull request, I'll get to it asap.

**Comment by otim** [comment]
Thanks!

File: plugin_description.xml [hunk 1]
```diff
@@ -253,6 +253,11 @@
       Makes it easy to move around a given point on the XY plane, looking at it from any angle.
     </description>
   </class>
+  <class name="rviz/ThirdPersonFollower" type="rviz::ThirdPersonFollowerViewController" base_class_type="rviz::ViewController">
+    <description>
+      Makes it easy to move around a given point on the XY plane, looking at it from any angle.
```
Comment (trainman419) [inline plugin_description.xml]: Please update the help text to reflect the fact that this controller also follow the orientation of the target frame.

**Comment by otim** [comment]
Addressed @trainman419 's comment

**Comment by wjwwood** [comment]
Thanks for the review @trainman419. @otim I'll try to get to this today or tomorrow.

**Comment by wjwwood** [comment]
This is on the list for this Thursday's bug squashing, I'll hopefully get to it then.

**Comment by otim** [comment]
Any news @wjwwood ?

**Comment by wjwwood** [comment]
Sorry, we didn't get to it last time, I'll try to spend sometime on it this week. Thanks for your patience.

**Comment by wjwwood** [comment]
Ok, I finally got around to testing this out. It works great! Thanks for the contribution @otim, and thanks for your patience. I'll release it into Indigo and Jade today.

**Comment by otim** [comment]
Thanks! That's great!

**Comment by wjwwood** [comment]
@otim if you have a chance can you add your view controller to the list here: http://wiki.ros.org/rviz/UserGuide#The_different_camera_types?

**Comment by otim** [comment]
Sure!

**Comment by otim** [comment]
I noticed that XY Orbit was also missing in the user guide. I added both now. Please check if you agree with my descriptions.

**Pull request merged**